lib/AcqList/hooks/useFiltersToogle/useFiltersToogle.test.js

```diff
@@ -0,0 +1,50 @@
+import { useLocalStorage, writeStorage } from '@rehooks/local-storage';
+import { act, renderHook } from '@testing-library/react-hooks';
+
+import {
+  useFiltersToogle,
+} from './useFiltersToogle';
+
+jest.mock('@rehooks/local-storage', () => ({
+  useLocalStorage: jest.fn(),
+  writeStorage: jest.fn(),
+}));
+
+describe('useFiltersToogle', () => {
+  it('should initial state from local storage', () => {
+    useLocalStorage.mockClear().mockReturnValue([true]);
+
+    const { result } = renderHook(() => useFiltersToogle('stripes-acq'));
+
+    const { isFiltersOpened } = result.current;
+
+    expect(isFiltersOpened).toBeTruthy();
+  });
+
+  it('should change value to oppotise when toggleFilters is called', () => {
```

Comment:
opposite?

Reply:
yes, copy paste from another place :) will fix it there as well